### PR TITLE
Fix movielens example query for user id 0

### LINKED ISSUE #85: movielens: invalid user id
The dataset doesn't have `user 0`, but this script queries it. 
https://github.com/jubatus/jubatus-example/blob/master/movielens/python/ml_analysis.py#L12

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix #85.

I also removed compiled Java class files.

### CODE REVIEW

File: movielens/cpp/ml_analysis.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ const string NAME = "recommender_ml";
 int main(int argc, char* argv[]) {
   jubatus::recommender::client::recommender r("localhost", 9199, NAME, 5);
 
-  for (int i = 0; i < 943; i++) {
+  for (int i = 1; i < 943; i++) {
```
Comment (TkrUdagawa) [inline movielens/cpp/ml_analysis.cpp]: need to `i <= 943`

File: movielens/python/ml_analysis.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
     recommender = client.Recommender("127.0.0.1", 9199, 'movie_len')
 
-    for i in range(0,943):
+    for i in range(1,943):
```
Comment (TkrUdagawa) [inline movielens/python/ml_analysis.py]: need to `range(1, 944)`

**Comment by kmaehashi** [comment]
@TkrUdagawa Thanks! Fixed.

**Comment by TkrUdagawa** [comment]
Thanks

**Pull request merged**